statemachine/callbacks.py

```diff
@@ -20,7 +20,7 @@ class CallbackPriority(IntEnum):
     AFTER = 40
 
 
-def allways_true(*args, **kwargs):
+async def allways_true(*args, **kwargs):
     return True
 
```

Comment:
This method is placed [as default behaviour on a predicate chain](https://github.com/fgmacedo/python-statemachine/blob/macedo/async-sm/statemachine/callbacks.py#L104) that now is awaitable... so the caller is not aware that this default is being used, or other method that was user defined.

So I think that I cannot turn it back to sync, otherwise I need to add logic to the caller to inspect if it has a coroutine or a normal method.